src/AvaloniaEdit/Text/TextLineRun.cs

```diff
@@ -58,11 +58,22 @@ public double Height
                     var box = embeddedObject.ComputeBoundingBox();
                     return box.Height;
                 }
-                // add 2pt for underline
-                return FontSize * HeightFactor + 2;
+                
+                return GetDefaultLineHeight(FontSize);
             }
         }
 
+        public static double GetDefaultLineHeight(double fontSize)
+        {
+            // add 2pt for underline
+            return fontSize * HeightFactor + 2;
+        }
+
+        public static double GetDefaultBaseline(double fontSize)
+        {
+            return fontSize * BaselineFactor;
```

Comment:
We should use the FontMetrics here instead of using hard codes numbers.

https://github.com/AvaloniaUI/Avalonia/blob/7842883961d094e08e9def7f30cf32fd573179c7/src/Avalonia.Visuals/Media/TextFormatting/FontMetrics.cs